storage-dev/migrations/2_white_listing.js

```diff
@@ -1,106 +0,0 @@
-const fs = require("fs");
```

Comment:
those files don't exist in the original SC repo, so we can't delete them.

Reply:
Aaaah I see... fair enough, I'll revert this change